File: go-sdk/pkg/models/shared/metadatacascadepolicies.go
package shared

// MetadataCascadePolicies
// The part of an API response that describes marker
// based pagination
type MetadataCascadePolicies struct {
	Entries    []MetadataCascadePolicy `json:"entries,omitempty"`
	Limit      *int64                  `json:"limit,omitempty"`
	NextMarker *int64                  `json:"next_marker,omitempty"`
	PrevMarker *int64                  `json:"prev_marker,omitempty"`
}

File: sdks/ts/src/sdk/models/shared/metadatacascadepolicies.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import { MetadataCascadePolicy } from "./metadatacascadepolicy";



// MetadataCascadePolicies
/** 
 * The part of an API response that describes marker
 * based pagination
**/
export class MetadataCascadePolicies extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=entries", elemType: MetadataCascadePolicy })
  entries?: MetadataCascadePolicy[];

  @SpeakeasyMetadata({ data: "json, name=limit" })
  limit?: number;

  @SpeakeasyMetadata({ data: "json, name=next_marker" })
  nextMarker?: number;

  @SpeakeasyMetadata({ data: "json, name=prev_marker" })
  prevMarker?: number;
}
